client/app/services/service-details/service-details.html

```diff
@@ -347,24 +347,28 @@ <h2 translate>Resources</h2>
                             </pf-list-view>
                         </div>
                     </li>
-                    <li class="list-group-item service-details-resource-group-item"
-                        ng-repeat="genericObject in vm.genericObjects">
-                        <a class="service-details-resource-group-title"
-                           ng-class="{'collapsed': !genericObject.isExpanded}"
-                           ng-click="vm.toggleOpenGenericObjects(genericObject)">
-                            {{genericObject.name}}
-                        </a>
-                        <div class="service-details-resource-list-container"
-                             ng-class="{'collapse': !genericObject.isExpanded}">
-                            <generic-objects-list generic-object="genericObject"
-                                                  on-update="vm.setGenericObjectViewState(object)"></generic-objects-list>
-                        </div>
-                    </li>
-                </ul>
+                </ul>      
             </section>
+         
         </div>
     </div>
-
+    <div ng-if="vm.genericObjects.length > 0">
```

Comment:
`<div ng-if="vm.genericObjects.length">`
will be true when anything is greater than 0